.eslintrc.json

```diff
@@ -0,0 +1,186 @@
+{
+    "env": {
+        "browser": true,
+        "node": true,
+        "jquery": true
+    },
+    "globals": {
+      "process.env": true
+    },
+    "root": true,
+    "rules": {
+        // Possible Errors
+        "comma-dangle": [2, "never"],
+        "no-cond-assign": 2,
+        "no-console": 1,
+        "no-constant-condition": 2,
+        "no-control-regex": 2,
+        "no-debugger": 2,
+        "no-dupe-args": 2,
+        "no-dupe-keys": 2,
+        "no-duplicate-case": 2,
+        "no-empty-character-class": 2,
+        "no-empty": 2,
+        "no-ex-assign": 2,
+        "no-extra-boolean-cast": 2,
+        "no-extra-parens": 2,
+        "no-extra-semi": 2,
+        "no-func-assign": 2,
+        "no-inner-declarations": 2,
+        "no-invalid-regexp": 2,
+        "no-irregular-whitespace": 2,
+        "no-negated-in-lhs": 2,
+        "no-obj-calls": 2,
+        "no-regex-spaces": 2,
+        "no-sparse-arrays": 2,
+        "no-unexpected-multiline": 2,
+        "no-unreachable": 2,
+        "use-isnan": 2,
+        "valid-jsdoc": [2, { "requireReturn": false }],
+        "valid-typeof": 2,
+
+        // Best Practices
+        "accessor-pairs": 2,
+        "block-scoped-var": 2,
+        "complexity": [2, 5],
```

Comment:
would probably raise it to 7-10
https://en.wikipedia.org/wiki/Cyclomatic_complexity#Limiting_complexity_during_development